plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/ext/provider/TerminalVolumeProvider.java

```diff
@@ -34,28 +34,41 @@
 @Singleton
 public class TerminalVolumeProvider implements Provider<String> {
 
-    private static final String CONTAINER_TARGET = ":/mnt/che/terminal:ro,Z";
+    private static final String RO_Z = "ro,Z";
+	private static final String CONTAINER_TARGET = ":/mnt/che/terminal";
     private static final String TERMINAL         = "terminal";
     private static final Logger LOG              = LoggerFactory.getLogger(TerminalVolumeProvider.class);
 
     @Inject
     @Named("che.workspace.terminal_linux_amd64")
     private String terminalArchivePath;
 
+    @Inject
+    @Named("che.machine.docker.volumes.workspace.options")
+    private String volumeOptions;
+
     @Override
     public String get() {
+        if (volumeOptions == null) {
```

Comment:
same here, as it's a required parameter it needs to be provided so we don't have to handle this